spec/jobs/socure_shadow_mode_proofing_job_spec.rb

```diff
@@ -349,22 +349,59 @@
       job.build_applicant(encrypted_arguments:, user_email:)
     end
 
-    it 'builds an applicant structure that looks right' do
-      expect(build_applicant).to eql(
-        {
-          first_name: 'FAKEY',
-          last_name: 'MCFAKERSON',
-          address1: '1 FAKE RD',
-          address2: nil,
-          city: 'GREAT FALLS',
-          state: 'MT',
-          zipcode: '59010-1234',
-          phone: '12025551212',
-          dob: '1938-10-06',
-          ssn: '900-66-1234',
-          email: user.email,
-        },
-      )
+    let(:expected_attributes) do
+      {
+        first_name: 'FAKEY',
+        last_name: 'MCFAKERSON',
+        address1: '1 FAKE RD',
+        address2: nil,
+        city: 'GREAT FALLS',
+        state: 'MT',
+        zipcode: '59010-1234',
+        dob: '1938-10-06',
+        ssn: '900-66-1234',
+        email: user.email,
+      }
+    end
+
+    context 'when the user has a phone directly passed in' do
+      # existing test
+      it 'builds an applicant structure that looks right' do
```

Comment:
'that looks right' is copypasta but now that there are several, I'd like to make the difference clearer.